contracts/azorius/LinearERC20VotingExtensible.sol

```diff
@@ -0,0 +1,327 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity =0.8.19;
+
+import {IVotes} from "@openzeppelin/contracts/governance/utils/IVotes.sol";
+import {BaseStrategy, IBaseStrategy} from "./BaseStrategy.sol";
+import {BaseQuorumPercent} from "./BaseQuorumPercent.sol";
+import {BaseVotingBasisPercent} from "./BaseVotingBasisPercent.sol";
+
+/**
+ * An [Azorius](./Azorius.md) [BaseStrategy](./BaseStrategy.md) implementation that
```

Comment:
Could you please update these comments to outline specific details about this `Extensible` strategy?

Reply:
sure!